test/e2e/form-validation.test.js

```diff
@@ -1,14 +1,126 @@
 const { assert, driver } = require('vl-ui-core').Test.Setup;
 const VlFormValidationPage = require('./pages/vl-form-validation.page');
+const { Key } = require('selenium-webdriver');
 
 describe('vl-form-validation', async () => {
-   const vlFormValidationPage = new VlFormValidationPage(driver);
+    const vlFormValidationPage = new VlFormValidationPage(driver);
+    
+    beforeEach((done) => {
+        vlFormValidationPage.load().then(() => {
+            done();
```

Comment:
gewoon de promise van load returnen is ook goed denk ik, beforeEach wacht op promisie die je er uit returnt dacht ik.